### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sdec18**
thanks and regards is imp bro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# t2
+# T2
```
Comment (Codesmaster-lab) [inline README.md:1]: good

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
 just a learning repo for git commands
+`Thanks and regards`
```
Comment (Codesmaster-lab) [inline README.md:3]: Aree srta abar code a kno?

**Review by Codesmaster-lab — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sdec18) [reply README.md:1]: thnx

**Review by Codesmaster-lab — APPROVED** [review]
baah

**Comment by Codesmaster-lab** [comment]
no more fajlami here

**Comment by Codesmaster-lab** [comment]
ok atlast made some serious changes

**Pull request merged**